Cargo.toml

```diff
@@ -49,9 +49,11 @@ unicode-width = "0.1.9"
 url = "2.2.2"
 
 [dependencies.reqwest]
-version = "0.11.18"
+git = "https://github.com/seanmonstar/reqwest"
```

Comment:
Wait for the new version of reqwest to be released, the current version 0.12.1 has 2 failing tests

Reply:
Reqwest just released a new version 0.12.2, so this PR can be ready to go